src/components/Datatable/ControlsModule/ControlsModule.tsx

```diff
@@ -194,7 +194,7 @@ const ControlsModule = <D extends Record<string, unknown>>({
   };
 
   return (
-    <FlexContainer flexDirection="column">
+    <Stack>
```

Comment:
I think that we can use the plain `div` element, there seems to be no benefit from using LP